server/server-config.js

```diff
@@ -218,6 +221,26 @@ app.post('/removeSavedTrip', (req, res) => {
   }
 });
 
+app.post('/zip', (req, res) => {
+  console.log('this is the SMS bodyyy!!!!!', req.body.Body);
+  const twilio = require('twilio');
+  let twiml = new twilio.TwimlResponse();
+  let zipCode = zip.cleanUserInputAsZipcode(req.body.Body);
+
+  Promise.resolve((zip.getWeatherForecast(zipCode))
+    .then( (results) => {
+      console.log('---11111-----', results);
+      twiml.message(results);
+
+      res.writeHead(200, {'Content-Type': 'text/xml'});
+      res.end(twiml.toString());
+    })
+    .catch( (err) => {
+      console.log('err', err);
```

Comment:
better error handling is needed. What response do you need to send if/when there is an explicit error?